src/Famix-Traits/FamixTWithTemplates.trait.st

```diff
@@ -31,8 +31,6 @@ FamixTWithTemplates >> templates [
 
 { #category : #accessing }
 FamixTWithTemplates >> templates: anObject [
-
-	<generated>
-	templates value: anObject
+	templates add: anObject
```

Comment:
This should be a new method #addTemplate: and not change the implementation of #templates: